ansible/playbooks/do_setup.yml

```diff
@@ -223,6 +223,16 @@
       register: do_registry_result
       when: use_do_registry
       failed_when: do_registry_result.stdout_lines | length < 2
+      ignore_errors: true
+
+    - name: d_ocean | registry | create registry endpoint
+      ansible.builtin.command: doctl registry create {{ registry_name }} --region {{ droplet_region }}
+      when: do_registry_result.stdout_lines | length < 2
+
```

Comment:
Should these also have a `when: use_do_registry`?

Reply:
Yes they should!

Reply:
Should that be added before we merge then? :)

Reply:
Yes absolutely. I have the changes, but I didn't test them yet so I didn't push them.